guides/common/modules/con_troubleshooting-katello.adoc

```diff
@@ -0,0 +1,26 @@
+[id="troubleshooting-katello"]
```

Comment:
My only concern here is that I don't want the ID to change later on. @maximiliankolb thoughts?

Reply:
Is this a special case where we don't want the ID to change?

Other than that, I recommend the following:
```suggestion
[id="Troubleshooting_Katello_{context}"]
```

Reply:
Well, as I said in https://github.com/theforeman/foreman-documentation/pull/1163#pullrequestreview-918448100 I think we should generalize it to just "troubleshooting".

Reply:
All three versions are basically fine for me; right now, I have no strong preference (because we don't include this file in downstream docs).